src/app/components/protests/my-protests/my-protests.component.html

```diff
@@ -0,0 +1,131 @@
+<ion-grid fixed="true" class="overflow-x-hidden overflow-y-auto">
+  <h1 class="b f2 mt1 pt1 mb0 pb0">My Protests</h1>
+  <h3 class="f6 mt0 pt0 black-70">
+    View protests you are organizing and or attending.
+  </h3>
+
+  <ion-row>
+    <section class="w-100">
+      <ion-col size-md class="pa0">
+        <h2 class="f3 mb0 fw6 ion-text-left pt0">Active Protests</h2>
+
+        <!-- chips -->
+        <div class="pb1">
+          <ion-chip
+            (click)="onClickChip(0)"
+            [ngClass]="
+              chipViewCondition == 0 ? 'bg-app-light-green' : 'default'
```

Comment:
these `0, 1, 2` values could possibly be a `string literal` instead. we can look at it on call.